src/components/case/CreateForm.vue

```diff
@@ -160,6 +173,36 @@ export default {
     removeTag(index) {
       this.tags.splice(index, 1);
     },
+    createCase(){
+      const {
+        title,
+        description,
+      }= this;
+       const { dispatch } = this.$store;
+
+      if (!title){
+        dispatch(
+            'alertStore/error',
+            stringFormat(config.messagingConfig.messages.error.field_error, 'Title').trim(),
+            { root: true }
+        );
+        return;
+      }
+
+       if (!description){
+        dispatch(
+            'alertStore/error',
+            stringFormat(config.messagingConfig.messages.error.field_error, 'Description').trim(),
+            { root: true }
+        );
+        return;
+      }
+      const createCase = {
+        title,
+        description,
+      }
+       dispatch('caseStore/createCase', createCase)
```

Comment:
invoke service directly rather dispatching store function. Refer to notifications.